vpn/main.cc

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include <windows.h>
+
+#include "base/at_exit.h"
+#include "base/command_line.h"
+#include "base/logging.h"
+#include "base/process/memory.h"
+#include "base/win/process_startup_helper.h"
+#include "brave/vpn/service_main.h"
+
+int main(int argc, char* argv[]) {
```

Comment:
we need to setup crash reporting for this

Reply:
Added crash reporting to `(channel)\User Data\Crashpad\reports`